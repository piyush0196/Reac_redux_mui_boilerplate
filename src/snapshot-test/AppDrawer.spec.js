import AppDrawer from 'components/ui-kit/AppDrawer'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

describe('Component - AppDrawer', () => {
  const wrapper = shallow(
    <AppDrawer
      openDrawer
      handleDrawer={() => { }}
      text='Demo Heading'
      isMobileView
    >
      <h4>Pass the list as a children in AppDrawer component</h4>
    </AppDrawer>
  )

  it('renders AppDrawer component correctly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
