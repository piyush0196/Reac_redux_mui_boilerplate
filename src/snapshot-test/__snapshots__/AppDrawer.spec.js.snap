// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - AppDrawer renders AppDrawer component correctly 1`] = `
<Styled(ForwardRef(Drawer))
  anchor="left"
  open={true}
  variant="temporary"
>
  <ForwardRef(Box)
    sx={
      Object {
        "display": "flex",
        "justifyContent": "space-between",
      }
    }
  >
    <Styled(div)>
      <CustomTypography
        align="inherit"
        display="initial"
        noWrap={false}
        sx={
          Object {
            "paddingLeft": "15px",
          }
        }
        value="Demo Heading"
        variant="h4"
      />
    </Styled(div)>
    <Styled(ForwardRef(IconButton))
      onClick={[Function]}
      padding="0px"
    >
      <ForwardRef(SvgSideArrow) />
    </Styled(ForwardRef(IconButton))>
  </ForwardRef(Box)>
  <ForwardRef(Box)
    sx={
      Object {
        "overflow": "auto",
      }
    }
  >
    <ForwardRef(Divider) />
    <h4>
      Pass the list as a children in AppDrawer component
    </h4>
  </ForwardRef(Box)>
</Styled(ForwardRef(Drawer))>
`;
